src/components/dms/ConvoMenu.tsx

```diff
@@ -52,31 +52,26 @@ let ConvoMenu = ({
   const leaveConvoControl = Prompt.usePromptControl()
   const {mutate: markAsRead} = useMarkAsReadMutation()
 
+  const {data: convo} = useConvoQuery(convoIdOrConvo)
```

Comment:
This might be a reason to handle `ConvoView` in the `Convo` agent so that they're always in sync. Not sure it matters to much rn, but it might in the future